### PR TITLE
Tiff support

### PR DESCRIPTION
**Pull request by robchia**
Dropzone doesn't handle TIFF images, so this is a workaround to getting TIFF thumbnails displayed.

The main idea is:

1. Convert TIFF -> PNG
2. Call `dropzone.createThumbnailFromUrl` with the converted image

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -54,18 +55,27 @@ class DropzoneController extends Controller {
...
         this.dropzone.removeFile(file);
         return;
       }
 
-      this.dispatchEvent(this.fileAddedEvent, { detail: { file: file }});
-
-      const controller = new DirectUploadController(this, file);
-      this.queue.push(controller);
-      this.runUploadQueue();
+      const isTiff = file.type === "image/tiff";
```
Comment (trobrock) [inline src/index.js:73]: does this need to be a variable or can you put this right in the conditional?

File: src/index.js [hunk 2]
```diff
@@ -115,6 +125,33 @@ class DropzoneController extends Controller {
     }
   }
 
+  createTiffThumbnail(file, callback) {
+    const self = this;
```
Comment (trobrock) [inline src/index.js:129]: Why do you need this? Shouldn't the method below already be bound to this properly?

**Review by trobrock — CHANGES_REQUESTED** [review]

File: src/index.js (hunk 1 above)
Comment (robchia) [reply src/index.js:73]: It's a habit to extract conditionals to easier to read, but I can remove the var here

Comment (trobrock) [reply src/index.js]: I like that strategy when it's not immediately clear from the conditional, this one basically reads the same to me though.

**Review by trobrock — APPROVED** [review]

**Review by tobyond — APPROVED** [review]
👍

**Pull request merged**